content/definition.md

```diff
@@ -8,7 +8,7 @@ Software is considered **Ethical Open Source** (EOS) when it meets *all* of thes
 
 1. **It benefits the commons.** The license under which the software is released must not prohibit modification, derivative works, and linking or compiling with other software (unless under a copyleft provision), in the general spirit of open source distribution.
 2. **It is created in the open.** The source code must be publicly available, developed and maintained in public view, and welcoming of public contributions (subject to review and approval by the software project’s maintainers.)
-3. **Its community is welcoming and just.** The software project’s community of maintainers and contributors must be governed by a comprehensive code of conduct that is consistently and fairly enforced.
+3. **Its community is welcoming and just.** The software project’s community of maintainers and contributors must adopt clear rules for decision-making and implementation, along with a comprehensive code of conduct that is consistently and fairly enforced.
```

Comment:
Not sure about "implementation" here. Could we instead say "decision-making and governance"?

Reply:
Good question. I've been trying not to get circular (governed with governance!), and it's tricky. I tend to think of governance, also, as the policy-making process and implementation as the executive/implementation process. And to me a critical piece here should be that the governance rules should make clear *how* the code of conduct will be enforced and *by whom*.

How about "adopt clear rules of decision-making and authority"? 

Another thing is I want the governance to carry the spirit of "welcoming." What if we made it "clear, inclusive rules"? Or some other language that indicates that newcomers should have an opportunity to get involved in governance if they wish?